README.md

```diff
@@ -1,9 +1,101 @@
+You can also install the agent from the Lacework Console. For a listing of the different agent install methods, see [Agent Install Options](https://support.lacework.com/hc/en-us/sections/360002375733).
+
+## Install Prerequisites
+Complete the following prerequisites steps before starting any of the agent install procedures below.
+1. Download the release package ```release.tgz``` file from this Lacework GitHub Agent Release repository, where ```release``` is replaced by the agent release number.
+    1. In the right frame, click **Releases** to view all the available agent releases.  
+    2. Find a release and click ```release.tgz``` to download the file that contains the agent installers, where ```release``` is replaced by the agent release number.
+2. Unzip the ```release.tgz``` file into a temporary directory such as ```/tmp```.
+3. Using the Lacework Console, create an agent access token  by following the instructions in [Create Agent Access Tokens](https://support.lacework.com/hc/en-us/articles/360036425594).
+4. In the Lacework Console, copy the Access Token by clicking the **Copy to clipboard icon** icon.
+
+## Install Using the install.sh Script
+For single host installations, Lacework recommends using the installation script called ```install.sh```.
+1. Complete the [prerequisites](#install-prerequisites) steps.
+2. Set the ```token``` environment variable equal to the agent access token copied from the Lacework Console.
+   ```
+   $ export token=<copied_agent_access_token>
+   ```
+3. Use ```sed``` to replace ```$1``` with a valid agent access token in the ```install.sh``` file.
+   ```
+   $ sed -i.bak "s/ARG1=\$1/ARG1=${token}/g" /tmp/install.sh
+   ```
+4. Run the install.sh script to install the agent by following the instructions in [Run the Lacework Agent Installation Script](https://support.lacework.com/hc/en-us/articles/360005321273).
+
+## Install using the Lacework Agent Chef Recipe
+1. Complete the [prerequisites](#install-prerequisites) steps.
+2. Set the ```token``` environment variable equal to the agent access token copied from the Lacework Console.
+   ```
+   $ export token=<copied_agent_access_token>
+   ```
+3. Use ```sed``` to replace ```$1``` with a valid agent access token in the ```install.sh``` file.
+   ```
+   $ sed -i.bak "s/ARG1=\$1/ARG1=${token}/g" /tmp/install.sh
+   ```
+4. Unzip ```chef.tar.gz``` into a temporary directory such as ```/tmp/chef```.
+   ```
+   $ tar -xzf chef.tar.gz
+   ```
+5. Copy the ```install.sh``` file with the updated token to the appropriate directory.
+   ```
+   $ cp install.sh /tmp/chef/datacollector/files/default
+   ```
+6. Install the agent by following the instructions in [Install with Chef](https://support.lacework.com/hc/en-us/articles/360005321413).
 
-Latest Lacework Documentation are available <a href="https://support.lacework.com/hc/en-us" target="_blank">**here**</a>.
+## Install using Daemonset Deployment files for Kubernetes
```

Comment:
For k8s and docker swarm please point to the docker command to use to pull the release specific image listed on the releases page under Lacework Agent Docker image

Reply:
Added the following:
  On the **Releases** page of this site, find the appropriate ```docker pull``` command for a specific release under the **Lacework Agent Docker image** heading.